### ISSUE TITLE
Person endpoint is erroring out

### ISSUE THREAD

**Issue by alexkwolfe** [activeprospect/leadconduit-integration-clearbit#14]
```json
HTTP/1.1 500 Internal Server Error
Content-Type: application/json
Date: Tue, 01 Dec 2015 20:53:17 GMT
Status: 500 Internal Server Error
X-Content-Type-Options: nosniff
Content-Length: 94
Connection: Close
 {
  "error": {
    "type": "api_error",
    "message": "Sorry, something went wrong. We have been notified."
  }
}
```

**Comment by cgrayson** [comment]
Their [status page](http://status.clearbit.com/) said yesterday that there were problems with that endpoint. It says they're okay today, but I still see errors. [This lead](https://next.leadconduit.com/events/565f3e49b364dedf09b8c347?event_id=565f3e4ab364dedf09b8c354) got an error on the "company" endpoint, too. I'll contact them.

**Comment by cgrayson** [comment]
No… :confounded: that request looks bad. Think it's the `url` type. @jgrayson676, mind looking at this?

**Comment by cgrayson** [comment]
Looking more closely, wondering how this lookup ever worked without the `valueOf()` that will (probably) ultimately fix it, I believe this was broken by a change in the [clearbit module](https://www.npmjs.com/package/clearbit). Namely, [this change](https://github.com/clearbit/clearbit-node/commit/39b31707055366c483acec3b9898ccf1725b2ce6#diff-130802354fb9d27a2f15a7ce773ae2a6L19), which was published in their `1.2.1` release on 11/17. It was first deployed in LC production on 11/23 (per the `/releases/20151123192334` directory). 

Exhibit A: [this lead](https://next.leadconduit.com/events/564f688942545714703b4bff?event_id=564f688b42545714703b4c09&recipient_id=55b9366e73748fbbe0b7aaaf&type=recipient&start=2015-11-01&end=2015-11-30), which worked fine, on Nov. 20

Exhibit B: [that lead](https://next.leadconduit.com/events/5656521c452351ae7f7a4e54?event_id=5656521f452351ae7f7a4e63&recipient_id=55b9366e73748fbbe0b7aaaf&type=recipient&start=2015-11-01&end=2015-11-30), which did _not_, on Nov. 25

:deerstalker:
